oarepo_communities/ui/communities_components/semantic-ui/js/communities_components/components/CommunityInvitationsModal/CommunityInvitationsModal.jsx

```diff
@@ -0,0 +1,221 @@
+                aria-expanded={isOpen}
+                aria-haspopup="dialog"
+              />
+            }
+          >
+            <Modal.Header>
+              {i18next.t("Invite users to the {{communityTitle}} community", {
+                communityTitle: community.metadata.title,
+              })}
+            </Modal.Header>
+            <Modal.Content>
+              <Form>
+                <Form.Field>
+                  <TextAreaField
+                    fieldPath="membersEmails"
+                    required
+                    autoFocus
+                    label={
+                      <FieldLabel
+                        label={i18next.t("Members")}
+                        icon={
+                          validEmailsCount + invalidEmailsCount > 0 &&
+                          validEmailsCount ===
+                            validEmailsCount + invalidEmailsCount
+                            ? "check circle green"
+                            : ""
+                        }
+                      />
+                    }
+                    onChange={(e, { value }) =>
+                      handleChange(value, setFieldValue)
+                    }
+                  />
+                  {invalidEmailsCount > 0 && (
+                    <Label
+                      className="mt-0"
+                      pointing
+                      prompt
+                      content={`${i18next.t(
+                        "Invalid emails"
+                      )}: ${values.emails.invalidEmails.join(", ")}`}
+                    />
+                  )}
+                  <label className="helptext">
+                    {i18next.t(
+                      `Emails shall be provided on separate lines. 
+                    Acceptable formats are johndoe@user.com or Doe John <johndoe@user.com>. 
+                    Note that invitations shall be sent only to the valid email addresses.`
+                    )}
+                  </label>
+                </Form.Field>
+                <Form.Field required className="rel-mt-1">
+                  <FieldLabel label={i18next.t("Role")} />
+                  <List selection>
+                    {usersCanInvite.map((u) => (
+                      <List.Item
+                        key={u.name}
+                        onClick={() => {
+                          if (values.role === u.name) return;
+                          setFieldValue("role", u.name);
+                        }}
+                        active={values.role === u.name}
+                      >
+                        <List.Content>
+                          <List.Header>{u.title}</List.Header>
+                          <List.Description>{u.description}</List.Description>
+                        </List.Content>
+                      </List.Item>
+                    ))}
+                  </List>
+                  <ErrorLabel fieldPath="role" />
+                </Form.Field>
+                <RichInputField
```

Comment:
Can we use oarepo-ui rich editor component?

Reply:
@mirekys It is not possible unfortunately, due to the fact we don't have formConfig there.